src/test/java/com/okta/tools/aws/settings/ConfigurationTest.java

```diff
@@ -136,28 +146,23 @@ void addOrUpdateProfileToNewConfigFileWithCustomPrefix() throws IOException {
     void addOrUpdateDefaultProfileToNewConfigFile() throws IOException {
         Configuration initiallyEmpty = new Configuration(new StringReader(""));
 
-        // Small function to copy a section to a map so we can easily compare it
-        Function<Profile.Section, Map<String, String>> sectionToMap = section ->
-                section.entrySet().stream()
-                        .collect(Collectors.toMap(Map.Entry::getKey, Map.Entry::getValue));
-
```

Comment:
:art: moved this into Settings as it is generally useful and I wanted to hide the settings field to hide configuration2 from the rest of the client code.